src/utilities/new_versions.jl

```diff
@@ -247,6 +243,7 @@ function make_pr_for_new_version(
     end
 
     # Make a dir for our SSH PrivateKey which we will use only if it has been enabled
+    created_pr = nothing
```

Comment:
I've never really been a fan of this, but I feel like sometimes you need to?
I guess the alternative is to keep the `return nothing` at the end of the function, and a pre-mature `return new_pr`?

Doesn't really matter, idk if you have thoughts on it?

Reply:
I think because of all the do blocks, you'd need to chain the returns up through every do block which would be more messy